_posts/en/newsletters/2021-01-20-newsletter.md

```diff
@@ -0,0 +1,125 @@
+  to handle scripts more complicated than single-sig or multisig.  For
+  example, allowing a hardware wallet to handle in-channel LN payments
+  or payments made from a [vault][topic vaults].  His post does a good
+  job of describing various problems that current hardware wallets can't
+  handle, but he notes that necessary "changes may be very difficult".
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [Bitcoin Core 0.21.0][Bitcoin Core 0.21.0] is a the next major version
+  of this full node implementation and its associated wallet and other
+  software.  Major new features include support for Tor version 3 hidden
+  services using [version 2 `addr` messages][topic addr v2], the optional
+  ability to serve [compact block filters][topic compact block filters],
+  support for [signets][topic signet] (including the default signet
+  which has [taproot][topic taproot] activated).  It also offers
+  experimental support for wallets that natively use [output script
+  descriptors][topic descriptors].  For a complete list of changes, see
+  the [release notes][bcc 0.21.0 notes].
+
+- [Rust Bitcoin 0.26.0][] is a new release of this library.  Major new
+  features include support for signet, version 2 `addr` messages,
+  and several improvements to [PSBT][topic psbt] handling.  See its
+  [release notes][rb notes] for details.
+
+- [BTCPay Server 1.0.6.7][] in the second maintenance release to
+  [1.0.6.5][btcpay server 1.0.6.5] from last week, which added "support
+  [for] a subset of output descriptors in the wallet setup" (see the
+  *notable changes* section below).  Other features and bug fixes were
+  also included.
+
+- [C-Lightning 0.9.3rc2][c-lightning 0.9.3] is a release candidate for a
+  new minor version of this LN node.
+
+- [LND 0.12.0-beta.rc5][LND 0.12.0-beta] is the latest release candidate
+  for the next major version of this LN node.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19937][] signet mining utility FIXME:jnewbery
+
+- [LND #4917][] disables the use of [anchor outputs][topic anchor
+  outputs] by default, a feature that was planned to be released in the
+  upcoming 0.12.0-beta.  Advanced users can still opt-in to using
```

Comment:
I think it would be interesting to say why this major feature was reverted, if anyone has insight.

Reply:
I 100% agree.  I'm curious myself, but I didn't see any reason listed.

Reply:
I messaged Oliver at Lightning Labs for insight.

Reply:
I think the write-up here is fine as it is.